code/AssetLib/USD/USDLoaderImplTinyusdz.cpp

```diff
@@ -278,6 +401,8 @@ void USDImporterImplTinyusdz::verticesForMesh(
         std::vector<const tinyusdz::tydra::SkelNode *> skeletonNodes;
         skeletonNodes.push_back(&render_scene.skeletons[skeleton_id].root_node);
         for (int i = 0; i < skeletonNodes.size(); ++i) {
+            const tinyusdz::tydra::SkelNode *skeletonNode = skeletonNodes[i];
```

Comment:
Please remove this line; `skeletonNode` is unused and will break the build in environments which treat warnings as errors

Reply:
Thank you! Fixed!